_data/versions.yml

```diff
@@ -1,3 +1,57 @@
+v2.3:
+- title: v2.3.0-rc1
+  note: |
+    12 June 2017
+
+    ### Changes / New Features
+
+    - Various improvements which reduce Felix CPU usage.
+    - Introduces the Calico fanout agent, Typha, for better performance when using the Kubernetes datastore driver.
+    - Calico now implements the networking.k8s.io/NetworkPolicy API semantics when using the Kubernetes datastore driver.
+      - **Note**: Calico with etcd continutes to implement the extensions/v1beta1 NetworkPolicy API semantics - this change has no effect on etcd based Calico deployments. 
+      - **Note**: This represents a change in how existing Kubenretes NetworkPolicies are enforced by Calico.  To maintain existing behavior when upgrading, follow these steps:
```

Comment:
this sounds a bit confusing being right after the etcd note. Can we move this up or mention this is only for KDD?